libosmscout-client-qt/src/osmscout/AvailableMapsModel.cpp

```diff
@@ -18,6 +18,51 @@
 */
 
 #include <osmscout/AvailableMapsModel.h>
+#include <qt5/QtCore/qabstractitemmodel.h>
+#include <qt4/QtCore/qlist.h>
+
+
+MapProvider AvailableMapsModelMap::getProvider() const
+{
+  return provider;
+}
+
+size_t AvailableMapsModelMap::getSize() const
+{
+  return size;
+}
+
+QString AvailableMapsModelMap::getSizeHuman() const
+{
+  double num = size;
+  QStringList list;
+  list << "KiB" << "MiB" << "GiB" << "TiB";
```

Comment:
We already have an libosmscout function for this. See String.h ByteSizeToString().